python/cudf/cudf/_lib/orc.pyx

```diff
@@ -201,22 +53,24 @@ cpdef read_parsed_orc_statistics(filepath_or_buffer):
     cudf.io.orc.read_orc_statistics
     """
 
-    cdef parsed_orc_statistics parsed = (
-        libcudf_read_parsed_orc_statistics(make_source_info([filepath_or_buffer]))
+    parsed = (
+        plc.io.orc.read_parsed_orc_statistics(
+            plc.io.SourceInfo([filepath_or_buffer])
+        )
     )
 
-    cdef vector[column_statistics] file_stats = parsed.file_stats
-    cdef vector[vector[column_statistics]] stripes_stats = parsed.stripes_stats
+    file_stats = parsed.file_stats
+    stripes_stats = parsed.stripes_stats
 
     parsed_file_stats = [
-        _parse_column_type_statistics(file_stats[column_index])
-        for column_index in range(file_stats.size())
+        file_stat
+        for file_stat in file_stats
     ]
```

Comment:
This comprehension is a trivial recreation of the input list. We no longer need comprehensions here at all. We should be able to just use `parsed.file_stats`. Same thing for `parsed.stripe_stats`.

(If there's a `vector` to `list` conversion needed, we can just call `list(...)`.)

Reply:
Good call.